roles/ceph-infra/templates/logrotate.conf.j2

```diff
@@ -0,0 +1,12 @@
+/var/log/ceph/*.log {
+    rotate 7
+    daily
+    compress
+    sharedscripts
+    postrotate
+        killall -q -1 ceph-mon ceph-mgr ceph-mds ceph-osd ceph-fuse radosgw rbd-mirror || pkill -1 -x "ceph-mon|ceph-mgr|ceph-mds|ceph-osd|ceph-fuse|radosgw|rbd-mirror" || true
```

Comment:
Is this necessary? i don't see any process taking hold of the log files.

Reply:
trying to keep consistency with what is made for RPM based deployment https://github.com/ceph/ceph/blob/master/src/logrotate.conf

not sure it is really needed but it doesn't hurt anyway.